packages/indexer-native/package.json

```diff
@@ -46,7 +46,7 @@
     "clean": "rm -rf ./node_modules ./binary ./build ./coverage ./native/target ./native/artifacts.json ./native/index.node"
   },
   "dependencies": {
-    "@graphprotocol/common-ts": "2.0.1",
```

Comment:
I hope touching this package doesn't break anything...

Reply:
Before merging, I'll check if the images built out of this PR work as expected.
If not, we will leave `indexer-native` untouched.

Reply:
I've deployed images built out of this PR to our testnet, and both `indexer-service` and `indexer-agent` seem to be working as before. `indexer-native` build succeeded :)